test/controllers/ModulesSpec.scala

```diff
@@ -52,11 +63,10 @@ object ModulesSpec extends PlaySpecification with Mockito {
     }
   }
 
-  def createModules: (Modules, ModulesLookup, ModuleDao) = {
-    val lookup = mock[ModulesLookup]
-    val dao = mock[ModuleDao]
-    val modules = new Modules(lookup, dao)
+  def createModules(app: play.api.Application): (Modules, ModulesLookup, ModuleDao) = {
+    val lookup = app.injector.instanceOf[ModulesLookup]
```

Comment:
Because controllers will be extending `AbstractController` and then using`Action` builder from `ControllerComponents`. So, since we already have an instance of app, instead of mock every single component in `ControllerComponents`, I've replaced it by the "real" objects.